packages/beacon-node/src/api/impl/validator/index.ts

```diff
@@ -375,7 +400,7 @@ export function getValidatorApi({
         slot,
         parentBlockRoot,
         randaoReveal,
-        graffiti: toGraffitiBuffer(graffiti || ""),
+        graffiti: toGraffitiBuffer(graffiti || (await defaultGraffiti)),
```

Comment:
we should only apply the default graffiti if the validator client does not send any `graffiti` value

```suggestion
        graffiti: toGraffitiBuffer(graffiti ?? (await defaultGraffiti)),
```